Styles/footer (Mobile, Tablet, Desktop)

dawidmarek95x opened the pull request:
Dodałem style dla sekcji Footer pod wszystkie 3 wersje tj. Mobile, Tablet i Desktop

dawidmarek95x committed:
styles/footer (Mobile)

Dokonano stylizacji sekcji footer dla wersji Mobile

dawidmarek95x committed:
styles/footer (Tablet & Desktop)

Dokonano stylizacji sekcji footer dla wersji Tablet i Desktop

patryktomasztomaszek commented on src/sass/layout/_footer.scss:
```diff
@@ -0,0 +1,127 @@
+  }
+  
+  .footer__title--container {
+    max-width: 474px;
```
Napis łamie się, powiększyć minimum do 477px

dawidmarek95x replied:
Dzięki, faktycznie przeoczyłem to. Dawałem taki jak w layoucie i jakoś nie zauważyłem :)

patryktomasztomaszek commented on src/sass/layout/_footer.scss:
```diff
@@ -0,0 +1,127 @@
+}
+
+.footer__copyright--container {
+  border-top: 1px solid #E1E1E1;
```
Border nie wyświetla się na całej szerokości strony
Dodać do elementu atrybut `width: 100%;`

dawidmarek95x replied:
Problemem nie był width tylko dodana klasa container do diva z copy, przez niego linia też był trzymana

annaojdana commented:
Nie mam uwag.

dawidmarek95x committed:
bugfix/footer

Dodatkowa klasa container dla footer__copyright--container powodowała, że linia nie rozciągała się na całą szerokosć strony. Usunąłem ją.
